### ISSUE TITLE
Bad asso roles fetching

### ISSUE THREAD

**Issue by CdRom1** [jpennors/bobby#12]
Hi,

I'm currently a "responsable" and a "vice-president" in some associations and I can't add new items. There's probably an issue with the roles fetching from the portail.

If you want me to do so, I can investigate the code. Don't hesitate to ask for examples, ids and logs.

**Comment by jpennors** [comment]
Fixed ?

**Comment by CdRom1** [comment]
Not really. There's one asso (Clé d'UT) where I'm vice-président and the "+" button for adding items appears. But on two other associations (SiMDE and Rhizome, where I'm "Vice-Président" and "Responsable"), I only have a view access.

If you want to sneak on the prod, my portail user id is a0eaace0-3af6-11e9-a268-c1d50a3af224.

**Comment by CdRom1** [comment]
Anyways, maybe we could create a portal permission called "inventaire"? Or even maybe something more precise like "inventaire-voir", "inventaire-editer"? Or your app could manage its own access system: the président gets full access, and he grants accesses to specific users.

**Comment by CdRom1** [comment]
A little update: the permissions are fetched only for one of my assos. I.e. if I'm VP in asso A and asso B, I can only edit the inventory of asso A and make bookings in the name of asso A. (this is pretty inconvenient).

**Comment by NastuzziSamy** [comment]
@CdRom1 can you avoid sharing uuids ? Even if it is yours and not critical, it still is a leak of security
